helsingborg-stad/app-mitt-helsingborg
# Feat/co applicant modal

## Explain the changes you’ve made
Added the ability to add a co applicant when opening a new application

## Explain why these changes are made
A user needs to be able to add a co applicant when applying for a new application, and to make our work easier, we want the co applicant to be added on a case before the case is opened, hence these changes are made.

## Explain your solution
Added two new modals, one for opening a form and/or opening the add co applicant modal, and the addCoApplicant modal for entering a personal number for the co applicant.

Added API for adding a new co applicant.


## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Make sure you have a new application case in backend
4. Start a new application

## Tested environments

- [] Building the Application on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

## Screenshots
![simulator_screenshot_512D43D7-EA83-45B4-8553-B63B170DA7E6](https://user-images.githubusercontent.com/9610681/172556369-4c0aa279-bbaf-471d-92f4-9d58bd636bf5.png)

![simulator_screenshot_C4D15653-126B-47FA-A240-A6B5C4F43B4F](https://user-images.githubusercontent.com/9610681/172556406-7cc80d01-063d-4392-9f2c-f01e9aa24ead.png)
